src/components/Camera/index.js

```diff
@@ -44,32 +44,34 @@ type CameraCommonType = {
   subTitle: string,
   onUserMedia: Function,
   onUploadFallback: File => void,
-  onWebcamError: Function,
   onUserMedia: void => void,
   i18n: Object,
-  isFullScreen: boolean
+  isFullScreen: boolean,
+  handleWebcamError: Function
 }
 
 type CameraPureType = {
   ...CameraCommonType,
   onFallbackClick: void => void,
   faceCaptureClick: void => void,
+  cameraHeight: number | Object,
   webcamRef: React.Ref<typeof Webcam>,
 }
 
 // Specify just a camera height (no width) because on safari if you specify both
 // height and width you will hit an OverconstrainedError if the camera does not
 // support the precise resolution.
 const CameraPure = ({method, autoCapture, title, subTitle, onUploadFallback, onFallbackClick,
-  onUserMedia, faceCaptureClick, webcamRef, isFullScreen, onWebcamError, i18n}: CameraPureType) => (
+  onUserMedia, faceCaptureClick, webcamRef, isFullScreen, handleWebcamError, i18n, cameraHeight}: CameraPureType) => (
     <div className={style.camera}>
       <Title {...{title, subTitle, isFullScreen}} smaller={true}/>
       <div className={classNames(style["video-overlay"], {[style.overlayFullScreen]: isFullScreen})}>
         <Webcam
           className={style.video}
           audio={false}
-          height={720}
-          {...{onUserMedia, ref: webcamRef, onFailure: onWebcamError}}
+          height={0}
+          key={cameraHeight}
```

Comment:
key? o.O

Reply:
sorry, I already added some comments, but I didn't see you didn't add me for reviewer. I'm assuming the PR is not yet ready